custom_components/plugwise/climate.py

```diff
@@ -77,6 +77,8 @@ def __init__(
         self, api, coordinator, name, dev_id, loc_id, model, min_temp, max_temp
     ):
         """Set up the Plugwise API."""
+        self._enabled_default = True
+
```

Comment:
Where is the rest of the enabled_default code?

Reply:
I removed this line now, lets keep the conversation open - if I added that here (for some reason) the climate entity wasn't here at all, but I don't recall if that was due to enabled in the SmileGateway entity or it being here (maybe we should just add it back in again).

Reply:
When you give climate the same treatment as the other platforms, they become disable-able?
And when you leave out all enabled_default-code then the climate-entities are always there?

I'd say, no need for disabling climate-entities :)

Reply:
I'll give it a whirl tonight, for now your last line was sort-of the quickfix :P

Reply:
Lets stick to 'not disabling the climate entity' itself, it makes sense